### PR TITLE
BPUB-1068 added info cash telegram command

### PR DESCRIPTION
**Pull request by jurbanekgeneralbytes**

### CODE REVIEW

File: server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/examples/chat/InfoCommand.java [hunk 1]
```diff
@@ -71,12 +77,7 @@ public boolean processCommand(IExtensionContext ctx, IConversation conversation,
                 displayInfoAboutBalances = true;
             }else if ("terminals".equalsIgnoreCase(parameter) || "terminal".equalsIgnoreCase(parameter) || "t".equalsIgnoreCase(parameter)) {
                 displayInfoAboutTerminals = true;
-                while (parameters.hasMoreTokens()) {
-                    String serialNumber = parameters.nextToken();
-                    if (serialNumber.startsWith("BT") || serialNumber.startsWith("PS") || serialNumber.startsWith("bt") || serialNumber.startsWith("ps")) {
-                        serialNumbers.add(serialNumber);
-                    }
-                }
+                serialNumbers = redSerialNumbers(parameters);
```
Comment (generalbytes) [inline server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/examples/chat/InfoCommand.java]: redSerialNumbers to readSerialNumbers

File: server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/examples/chat/InfoCommand.java [hunk 2]
```diff
@@ -86,23 +87,94 @@ public boolean processCommand(IExtensionContext ctx, IConversation conversation,
             }else if (isPhoneNumber(parameter)) {
                 displayIdentityInformation(ctx, conversation, null, parsePhoneNumber(parameter));
                 return true;
+            } else if ("Cash".equalsIgnoreCase(parameter) || "cash".equalsIgnoreCase(parameter) || "c".equalsIgnoreCase(parameter)) {
```
Comment (generalbytes) [inline server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/examples/chat/InfoCommand.java]: Remove Cash

File: server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/examples/chat/InfoCommand.java (hunk 1 above)
Comment (jurbanekgeneralbytes) [reply server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/examples/chat/InfoCommand.java]: done

File: server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/examples/chat/InfoCommand.java (hunk 2 above)
Comment (jurbanekgeneralbytes) [reply server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/examples/chat/InfoCommand.java]: done

**Pull request merged**